Makefile

```diff
@@ -252,6 +259,9 @@ fixtures/debian-complex-dists: fixtures gnupghome
 fixtures/debian-missing-architecture: fixtures gnupghome
 	GNUPGHOME=$$(realpath -e gnupghome) debian/gen-missing-architecture-fixtures.sh $@
 
+fixtures/debian-flat: fixtures gnupghome
```

Comment:
Actually it may be a good test, since it's important to know that the signature is found in the correct location, and this repo format messes around with all the paths.
Also thanks to Make gnupghome is only set up once per stage container anyway.

Reply:
Done